spec/theoretically.it.spec.js

```diff
@@ -0,0 +1,130 @@
+var _ = require("lodash");
+var theoretically = require("../index.js");
+
+describe("A theory", function() {
+    var oldIt;
+
+    beforeEach(function() {
+        oldIt = it;
+    });
+
+    afterEach(function() {
+        it = oldIt;
+    });
+
+    it("runs `it` with each value in the value array and a formatted message", function() {
+        it = jasmine.createSpy("it");
+
+        var handler = jasmine.createSpy("handler");
+        theoretically.it("might run a test with %s as the value", [ "x", "y", "z" ], handler);
+
+        expect(it).toHaveBeenCalledWith("might run a test with x as the value", jasmine.any(Function));
+        expect(it).toHaveBeenCalledWith("might run a test with y as the value", jasmine.any(Function));
+        expect(it).toHaveBeenCalledWith("might run a test with z as the value", jasmine.any(Function));
+
+        var itCalledWithX = _.find(it.calls.allArgs(), function(args) {
+            return args[0] == "might run a test with x as the value";
+        });
+
+        var itCalledWithY = _.find(it.calls.allArgs(), function(args) {
+            return args[0] == "might run a test with y as the value";
+        });
+
+        var itCalledWithZ = _.find(it.calls.allArgs(), function(args) {
+            return args[0] == "might run a test with z as the value";
+        });
+
+        if(!itCalledWithX || !itCalledWithY || !itCalledWithZ) {
+            return expect("Failed to get one of the calls").toBeFalsy();
```

Comment:
This is to spot if something went wrong with the `_.find`. The idea is that consistency between the first `expect`s and the `_.find` is hard to guarantee. This check means that the test will fail instead of the testing framework crashing :smile: